agogosml/agogosml/common/flask_http_listener_client.py

```diff
@@ -17,7 +16,7 @@ def __init__(self, config: dict):
             PORT
             HOST
         """
-        self.port = int(config.get('PORT'))
+        self.port = int(config['PORT']) if 'PORT' in config else None
```

Comment:
config.get('PORT') returns None if PORT is not present. What's the advantage here of the ternary? Curious.

Reply:
If the environment variable is not set, `int(None)` will cause an exception:

![image](https://user-images.githubusercontent.com/1086421/52134438-8e95eb00-2611-11e9-91d3-113416629f4b.png)

Reply:
This is the type of bug that mypy finds for you.